src/ActuatorDiskFAST.C

```diff
@@ -0,0 +1,360 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2014 Sandia Corporation.                                    */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+/*
+ * ActuatorDiskFAST.C
+ *
+ *  Created on: Oct 16, 2018
+ *      Author: psakiev
+ */
+
+#ifdef NALU_USES_OPENFAST
```

Comment:
With the explicit listing of files in `CMakeLists.txt` I think we can clean up these `#ifdef`s in the C files.

Reply:
Yeah that's a good point.  They're pretty redundant.